docs/framework/get-started/system-requirements.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: ".NET Framework system requirements"
 description: "Find out what are the hardware, operating system and software requirements to install .NET Framework 4.5 and later versions."
-ms.custom: "updateeachrelease"
```

Comment:
Caught this while reviewing https://github.com/dotnet/docs/pull/20124

This article needs to be updated from time to time, but based on new Windows releases, not .NET Framework releases. Would you still use this metadata for that?

Reply:
It's safe to remove this one. The requirements are Windows based, not .NET based.